.gitignore

```diff
@@ -62,3 +62,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+openassessment/xblock/static/css/openassessment-ltr.css.map
```

Comment:
Or if you want to keep them, which may be a good idea, can you make this more general? Something like ``*.css.map`` makes sense.